services/web/src/lib/reusable-components/test-form.tsx

```diff
@@ -0,0 +1,44 @@
+'use client';
+
+import { zodResolver } from '@hookform/resolvers/zod';
```

Comment:
I really like that you made this is a separate component. Do not remove it as of now but before merging this PR we should remove this. Because the PR is only for creating the component itself.